assets/src/edit-story/app/media/utils/getResourceFromLocalFile.js

```diff
@@ -101,17 +104,23 @@ const getImageResource = async (file) => {
 const getVideoResource = async (file) => {
   const fileName = getFileName(file);
   const mimeType = file.type;
+
   const reader = await createFileReader(file);
-  const src = window.URL.createObjectURL(
-    new window.Blob([reader.result], { type: mimeType })
-  );
+
+  const src = createBlob(new Blob([reader.result], { type: mimeType }));
+
+  const videoEl = document.createElement('video');
+  const canPlayVideo = '' !== videoEl.canPlayType(mimeType);
+
   const frame = await getFirstFrameOfVideo(src);
```

Comment:
Why not wrap `getFirstFrameOfVideo` in a check if `canPlayVideo` is true?

Reply:
Funnily enough, `getFirstFrameOfVideo()` can still work, even though `canPlayVideo` is `false`.

So for MOV files with MP4 codec, Chrome will happily let us create a poster image, but the video itself cannot be played.

Hence no wrapping.

Reply:
Which is why I asked the question regarding the codec - https://github.com/google/web-stories-wp/pull/6541#discussion_r584801907